### ISSUE TITLE
isValidDecimalInput error message path

### ISSUE THREAD

**Issue by michiim** [chrishoermann/zod-prisma-types#100]
why do you define "path" for errors of isValidDecimalInput. I think it doesn't make sense because it affects the default behaviour. As a result, the error messages for these properties are not accessible in the same way as the messages for the other properties

export const MyModelSchema = z.object({
  id: z.number(),
  decimal: z
    .union([z.number(), z.string(), DecimalJSLikeSchema])
    .refine((v) => isValidDecimalInput(v), {
      message: 'Field "decimal" must be a Decimal',
      **path: ['Models', 'DecimalModel'],**
    }),
});

**Comment by chrishoermann** [comment]
@michiim true. I'll change it and add this hint to the model in the message string.

**Comment by chrishoermann** [comment]
@michiim I changed this behaviour in the latest release
